EdvClient.js

```diff
@@ -1089,7 +1086,14 @@ export class EdvClient {
     ]);
     delete encrypted.content;
     delete encrypted.meta;
-    delete encrypted.stream;
+
+    if(encrypted.stream) {
+      encrypted.stream = {
+        sequence: encrypted.stream.sequence,
+        chunks: encrypted.stream.chunks
+      };
+    }
+
```

Comment:
Since I don't think it makes sense to make this a special code block, I suggest we remove the whitespace. We're setting all the stuff on `encrypted` and `stream` just happens to need a bit of extra work, but it's the same thing as setting the other properties.

```suggestion
    if(encrypted.stream) {
      encrypted.stream = {
        sequence: encrypted.stream.sequence,
        chunks: encrypted.stream.chunks
      };
    }
```

Reply:
Got it, makes sense.